src/content/en/tools/chrome-devtools/beginners/devtools-for-beginners1.md

```diff
@@ -0,0 +1,239 @@
+to do this? 
+
+Enter DevTools.
+
+### A Note on Nodes {: #nodes}
+
+ When you open the Elements Panel in DevTools, 
+ you’ll see a screen that looks quite similar to the HTML document
+ you’ve been working on in the editing tab. 
+ However, these are not HTML elements, but are actually DOM nodes.
+ DOM is an interface that represents HTML elements in your browser, 
+ and while the DOM Tree you see in the Elements Panel looks quite 
+ similar to your HTML document now, 
+ there are ways you can edit it so that it is not. 
+
+1. Navigate to the viewing tab of your website.
+2. Open DevTools with <kbd>Command</kbd>+<kbd>Shift</kbd>+<kbd>I</kbd>.
+3. Directly under the `<script>` tag, you will see a new `<div>` element
+ that doesn't exist in your HTML document in the editing tab.
+ 
+ Using JavaScript, you can add nodes to the DOM Tree without HTML. 
+ This will play a bigger role when you learn about the Console 
+ panel and JavaScript, but for now, you’ll be editing a few DOM nodes 
+ yourself.
+ 
+### Edit DOM Nodes as HTML {: #edit-as-html}
+ The DevTools ‘Inspect Element’ and ‘Edit as HTML’ functions allow you 
+ to view changes you make to the DOM in real time.
+ You can try it out by seeing what it 
+ would look like to add some more content to your page! 
+ 
+ 1. Open devtools using 
+ <kbd>Command</kbd>+<kbd>Shift</kbd>+<kbd>I</kbd>. You should see 
+ something like this: 
+
+ <figure> <img src="elpanel.png" alt=" The Elements Panel Opened" width="auto" height="auto">
+<figcaption>
+    <b>Figure 4</b>. The elements panel.
+  </figcaption>
+ </figure>
+
+2. Right click on the `<About>` div, then click `Edit as HTML`. 
+The following screen will appear:
+
+<figure> <img src="editashtml.png" alt="Editing as HTML" width="auto" height="auto">
+<figcaption>
+    <b>Figure 5</b>. Editing an object as HTML.
+  </figcaption>
+</figure>
+
+Now you have a live view of your changes! 
+
+Try adding information to the About node:
+
+1. Add a paragraph element like: 
+`<p> My website is all about sharing my  achievements! </p>`. 
+You can do this by clicking under `<div class= “About”>` 
+and typing in your code.
+2. Add a header element like `<h4>Here’s my resume.</h4>` 
+under the paragraph element.
+3. Add a button under the header element with 
+`<button>Download it!</button>`
+4. Copy your code and paste it under line 13 inside the “About” `<div>`.
+
+Note: If you refresh the page or close the tab, 
+your edits will be gone forever. After all, 
+imagine what would happen if you could permanently change 
+the HTML and text of any website! So,
+make sure to copy any changes you make to your site in DevTools.
+
+If you know what you want to edit, then there’s an even simpler workflow:
+
+1. Open the viewing tab.
+2. Highlight a section of your website (like an image or line of text).
+3. Right click and choose **Inspect**. 
+4. Double click on either the content or tag of the element to edit it.
```

Comment:
Suggestion to make this an actual example, where they select some text, and then inspect it, and then change it-- and show the change too. It is possible to make this fun-- as they could open any webpage, including your hosted about me page, and add their own text.

Reply:
Done, but it's not really fun.